src/main/java/gov/usgs/cida/pubs/springinit/SecurityConfig.java

```diff
@@ -61,4 +66,14 @@ public void configure(HttpSecurity http) throws Exception {
 				.csrf().csrfTokenRepository(CookieCsrfTokenRepository.withHttpOnlyFalse())
 		;
 	}
+
+	@Bean
+	public CorsConfigurationSource corsConfigurationSource() {
+		CorsConfiguration configuration = new CorsConfiguration();
+		configuration.setAllowedOrigins(Arrays.asList("*"));
+		configuration.setAllowedMethods(Arrays.asList("GET","POST"));
```

Comment:
We currently allow
* methods 'POST, GET, OPTIONS, DELETE, PUT'
* headers 'Origin, Accept, X-Requested-With, Content-Type, Authorization'
 on prod.

(but have nothing set on dev...)

Reply:
I added options, delete, and put. Is it good to go?